app/tests/e2e/test_issuer.py

```diff
@@ -467,7 +467,7 @@ async def test_revoke_credential(
         for rec in records
         if (
             rec["role"] == "issuer"
-            and rec["state"] == "credential-issued"
+            and (rec["state"] == "credential-issued" or rec["state"] == "done")
```

Comment:
@lohanspies had to make this minor change to the test. `credential-issued` does not always appear in the records state, and may also be in state "done"